appengine/flexible/endpoints/README.md

```diff
@@ -57,9 +57,9 @@ With the API key, you can use the echo client to access the API:
 
     $ python clients/echo-client.py https://YOUR-PROJECT-ID.appspot.com YOUR-API-KEY
 
-### Using the JWT client.
+### Using the JWT client (with key file)
```

Comment:
All the three auth samples added here are to show how to authenticate with endpoints without a key file.